### ISSUE TITLE
Compiler Error

### ISSUE THREAD

**Issue by KenIsStupid** [genesistoxical/drop-icons#3]
When I press compile on the installer.iss, a compiler error pops up saying, "Line 49: No found matching "C:\Users\{...}\Downloads\drop-icons-main\drop-icons-main\installer src\,,\src\DropIcons\bin\Release\*." The Output file still shows, but is empty.

**Comment by genesistoxical** [comment]
Have you already compiled the project in Visual Studio? The installer probably didn't find the binaries if they haven't been compiled, that's what "**...bin\Release**" says.

**Comment by genesistoxical** [comment]
@ImMaxRoss Thank you for providing details of the issue.

> The generated "Drop-Icons-Installer.exe" seems to run successfully but the new "Drop Icons" directory does not include the "Drop Icons.exe" file after the installation.

Yes, I assumed from the first comment that it was because the executable file didn't exist yet, it can be created by compiling the code in the "**src**" folder. What happens is that I uploaded the compiled binaries in [Releases](https://github.com/genesistoxical/drop-icons/releases) (portable and installable version), as well as on the [page](https://genesistoxical.github.io/drop-icons/) you mention.

In short, you must compile the application itself (**Drop Icons.exe**) with _Visual Studio_ and then compile the installer with _Inno Setup_.

I'll add details in the **README** about this so there is no confusion. Thank you very much once again for your contribution. 😊

**Comment by genesistoxical** [comment]
Information about this issue has been added to the **README**, as it is not actually a bug. ✅
